examples/compute/cuda/hello_compute.cu

```diff
@@ -60,7 +60,8 @@ int hpx_main(boost::program_options::variables_map& vm)
             int i = blockDim.x * blockIdx.x + threadIdx.x;
             if(i < N)
                 C[i] = A[i] + B[i];
-        }, d_A.data(), d_B.data(), d_C.data());
+        }, (int*)(d_A.data()), (int*)(d_B.data()), (int*)(d_C.data())
+        );
 
```

Comment:
The problem I'm referring to is that clang apparently instantiates [this function](https://github.com/STEllAR-GROUP/hpx/blob/master/hpx/compute/cuda/detail/launch.hpp#L67-L71) in host code, which leads to the compilation error you're trying to work around.